ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -256,6 +256,10 @@ objects:
         value: openstack-cloud
       - name: OS_CLIENT_CONFIG_FILE
         value: /etc/openshift-installer/clouds.yaml
+      - name: USER
+        value: test
+      - name: HOME
+        value: /home/test
```

Comment:
Do we have permission to create this?  In #1185, I'd just set `HOME` to `/tmp`.

Reply:
mmh, good point. I was gonna set it to tmp and then changed my mind!